FlashFeetAPI/src/main/java/org/example/trueApplication.java

```diff
@@ -0,0 +1,38 @@
+package org.example;
+
+import io.dropwizard.Application;
+import io.dropwizard.setup.Bootstrap;
+import io.dropwizard.setup.Environment;
+import io.federecio.dropwizard.swagger.SwaggerBundle;
+import io.federecio.dropwizard.swagger.SwaggerBundleConfiguration;
+
+public class trueApplication extends Application<trueConfiguration> {
+
+    public static void main(final String[] args) throws Exception {
+        new trueApplication().run(args);
+    }
+
+    @Override
+    public String getName() {
+        return "true";
+    }
+
+    @Override
+    public void initialize(final Bootstrap<trueConfiguration> bootstrap) {
+        // TODO: application initialization
+        bootstrap.addBundle(new SwaggerBundle<trueConfiguration>(){
+            @Override
+            protected SwaggerBundleConfiguration getSwaggerBundleConfiguration(trueConfiguration configuration) {
+                return configuration.getSwagger();
+            }
+        });
+    }
+
+    @Override
+    public void run(final trueConfiguration configuration,
+                    final Environment environment) {
+        // TODO: implement application
+            //environment.jersey().register( <ControllerName> );
```

Comment:
Again, commented code would not be allowed to be merged in unless for a really good reason. Also this PR claims to complete US001 but has no controllers registered. Sounds like an issue? 🙃